### ISSUE TITLE
Support verified android App Links

### ISSUE THREAD

**Issue by DeD1rk** [svthalia/concrexit#2679]
Android introduced something similar to apples AASA files: `thalia.nu/.well-known/assetlinks.json`.

We need to host this there:
```
[
  {
    "relation": [
      "delegate_permission/common.handle_all_urls"
    ],
    "target": {
      "namespace": "android_app",
      "package_name": "com.thaliapp",
      "sha256_cert_fingerprints": [
        "25:F7:06:CA:04:36:32:62:87:10:7C:59:84:5E:A0:E7:68:A4:DD:80:E1:CF:EB:30:96:3C:B1:CC:CF:E4:32:16"
      ]
    }
  }
]
```
with contenttype application/json.

Here is some context: https://developer.android.com/training/app-links/, https://play.google.com/console/u/0/developers/7534210298542017316/app/4976211217569319891/deeplinks?selectedVersionCode=616. The Play Console can verify whether it's been set up correctly.

**Comment by JAicewizard** [comment]
Could this also be implemented for the staging app-id on staging? I'm afraid (but not sure) that otherwise links to the staging app won't work

**Comment by DeD1rk** [comment]
Staging app isnt a different thing on the play console right? I guess we could add the same sha with differen package name but I'd prefer to stick with the stuff generated by the play console. If the play store won't generate anything for the debug app, it's probably fine to just not have the links verified for those builds. That won't break anything

**Comment by JAicewizard** [comment]
On the playstore there is no staging app, it doesn't know it exists so it also doesnt generate anything for it. We can just try to see if this still works for staging and try to fix if it breaks.

**Comment by ColonelPhantom** [comment]
Is this working or not? If it is then we can close this issue.

**Comment by JAicewizard** [comment]
I think so yeah, it worked last week

**Comment by JAicewizard** [comment]
This does not work, thalia.nu/.well-known/assetlinks.json is a dead link

**Comment by JAicewizard** [comment]
Probably not migrated from nix, as this was a nix config

**Comment by DeD1rk** [comment]
Fixed it in 30fab6a9 🤦

**Comment by JAicewizard** [comment]
Ah I see, easy error to make :) Now wait for release
